pkg/utils/watchChaosContainer.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"time"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GetChaosContainerStatus gets status of the chaos container
+func GetChaosContainerStatus(experimentDetails *ExperimentDetails, clients ClientSets) (bool, error) {
+
+	isCompleted := false
+	PodList, err := clients.KubeClient.CoreV1().Pods(experimentDetails.Namespace).List(metav1.ListOptions{LabelSelector: "job-name=" + experimentDetails.JobName})
+	if err != nil || len(PodList.Items) == 0 {
+		return false, errors.Wrapf(err, "Unable to get the chaos pod, due to error: %v", err)
+	}
+
+	for _, pod := range PodList.Items {
+		if pod.Status.Phase == corev1.PodRunning || pod.Status.Phase == corev1.PodSucceeded {
+			for _, container := range pod.Status.ContainerStatuses {
+
+				if container.Name == experimentDetails.JobName && container.State.Terminated != nil {
```

Comment:
Let's add the reason for container name (in this case the reason is because we construct the job that way in runner)